### DISCUSSION TITLE
Unwanted branding outside of app

### DISCUSSION THREAD

**Discussion by amandamana** [wstyres/Zebra#2005] [Q&A] [answered]
The Zebra icon appeared in the bottom left corner of my lock screen. The application switcher was also treated to a lower left badge and banner displaying "Good afternoon".

What's up with that?

Is this vanity patching considered an easter egg? I downgraded to 1.1.26, and the branding was still there initially, but then disappeared. Does it come and go? In what version did this first appear? Can it be disabled?

**Comment by iAdam1n** [comment]
I have never seen that before. Did you download it from the official repo, https://getzbra.com/repo, and if so, could you please upload a video showing what's happening?

**Comment by amandamana** [comment]
Yes, official repo. I haven't seen it before either. This was a newly jailbroken iPhone 5 (iOS 10.3.4), and since Zebra on my other installs is not up to date (e.g. v1.1.21 on my daily driver), I guessed it was a new feature.

I've re-upgraded back to 1.1.28 on the device that displayed the branding, and will take screenshots if and when it reoccurs.

**Comment by absidue** [comment, marked as answer]
This sounds like iOS's built in suggestions system. If it notices that you use the same apps at a similar time every day, it will start suggesting those apps at those times of day. You might notice a similar thing happening if you always use the same app for music when you are using bluetooth headphones, you will see a banner on the app switcher saying that you use that music app often with your headphones connected.

tl;dr, it's nothing to do with Zebra, it's the operating system analysing your behaviour and trying to be helpful. It's out of our control.

**Reply by kirb** [reply]
Yep this is what it is. Just to add on, seems like Apple agreed the bottom-left lock screen suggestion icon was kinda just confusing and not very useful, so it was removed in iOS 11. Seems to be much better at figuring out patterns as well, less likely to suggest something that doesn’t make much sense, like what might have happened here.

**Comment by amandamana** [comment]
[Mystery solved](https://osxdaily.com/2015/04/18/turn-off-suggested-apps-ios/)! I'm shocked that I haven't seen this before, which must be because I routinely disable it when setting up a device for regular use.
